src_common/uint256.c

```diff
@@ -240,7 +240,7 @@ bool tostring256(const uint256_t *const number,
 
     if (offset == outLength) {  // destination buffer too small
         if (outLength > 3) {
-            strcpy(out, "...");
+            memcpy(out, "...", 4);
```

Comment:
strlcpy with outLength as param feels better

Reply:
Fixed in [cc283f2](https://github.com/LedgerHQ/app-ethereum/pull/420/commits/cc283f271c398088f990c24601dcdaac060eb30b)